### PR TITLE
Add Medicaid eligibility

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
* Adds parameters from medicaidscreener, and logic

### CODE REVIEW

File: openfisca_us/variables/gov/hhs/medicaid/eligibility/medicaid_person_type.py [hunk 1]
```diff
@@ -25,12 +27,16 @@ def formula(person, period, parameters):
         has_dependents = person.spm_unit.any(
             person.spm_unit.members("age", period) < 19
         )
+        under_21_qualifies = parameters(period).hhs.medicaid.under_21_qualifies_as_child
+        state = person.household("state_code_str", period)
+        state_has_under_21_child_category = under_21_qualifies[state]
         return select(
-            [age == 0, age < 6, age < 19, has_dependents, True],
+            [age == 0, age < 6, age < 19, (age < 21) & state_has_under_21_child_category, has_dependents, True],
```
Comment (fedderw) [inline openfisca_us/variables/gov/hhs/medicaid/eligibility/medicaid_person_type.py:34]: Nice! I didn't understand how this would have worked before

Comment (nikhilwoodruff) [reply openfisca_us/variables/gov/hhs/medicaid/eligibility/medicaid_person_type.py:34]: Thanks, yeah I did wonder if there was a more elegant solution but this seems to work fine

**Comment by MaxGhenis** [comment]
https://github.com/PolicyEngine/openfisca-us/pull/443 in case useful

**Comment by nikhilwoodruff** [comment]
Todo:
- [ ] Add parameter for whether each state extends pregnancy coverage from 60 days postpartum to 12 months
- [ ] Integrate postpartum logic

File: openfisca_us/variables/gov/hhs/medicaid/eligibility/medicaid_person_type.py [hunk 2]
```diff
@@ -19,18 +21,38 @@ class medicaid_person_type(Variable):
     documentation = "Person type for Medicaid"
 
     def formula(person, period, parameters):
+        ma = parameters(period).hhs.medicaid
         # Get the person's age.
         age = person("age", period)
         # Get the existence of dependents, as defined by people 18 or younger.
         has_dependents = person.spm_unit.any(
             person.spm_unit.members("age", period) < 19
```
Comment (MaxGhenis) [inline openfisca_us/variables/gov/hhs/medicaid/eligibility/medicaid_person_type.py:29]: could you make this a policy parameter?

Comment (nikhilwoodruff) [reply openfisca_us/variables/gov/hhs/medicaid/eligibility/medicaid_person_type.py:29]: Yes, but it raises the question of how to label the FPL limit parameters and the category name

Comment (MaxGhenis) [reply openfisca_us/variables/gov/hhs/medicaid/eligibility/medicaid_person_type.py:29]: Oh I'm not suggesting changing those, just the age limit for defining dependents.

**Pull request merged**